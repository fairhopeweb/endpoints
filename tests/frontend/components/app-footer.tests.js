import { mount } from '@vue/test-utils';
import AppFooter from '@/components/app-footer.vue';

describe('AppFooter', () => {
    const wrapper = mount(AppFooter);

    test('is a Vue instance', () => {
        expect(wrapper.isVueInstance()).toBeTruthy();
    });

    test('matches snapshot', () => {
        expect(wrapper.element).toMatchSnapshot();
    });
});
