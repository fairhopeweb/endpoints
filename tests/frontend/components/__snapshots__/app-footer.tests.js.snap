// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter matches snapshot 1`] = `
<footer
  class="app-footer"
>
  <div
    class="wrapper"
  >
    <small>
      
            Developed by
            
      <a
        href="https://github.com/uncaughtxcptn"
        rel="noopener noreferrer"
        target="_blank"
      >
        Uncaught Exception
      </a>
    </small>
     
    <a
      class="github-link"
      href="https://github.com/uncaughtxcptn/endpoints"
      rel="noopener noreferrer"
      target="_blank"
    />
  </div>
</footer>
`;
